### PR TITLE
add the option to disable certificate verification

### PR DESCRIPTION
**Pull request by jsvd**
solves #160

### CODE REVIEW

File: lib/logstash/outputs/elasticsearch.rb [hunk 1]
```diff
@@ -216,6 +216,9 @@ class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
   # Enable SSL
   config :ssl, :validate => :boolean, :default => false
 
+  # Enable SSL
```
Comment (talevy) [inline lib/logstash/outputs/elasticsearch.rb]: can we update message here to be more descriptive to what this config does?

Comment (suyograo) [inline lib/logstash/outputs/elasticsearch.rb]: ++, this gets generated to the docs directly

File: spec/outputs/elasticsearch_spec.rb [hunk 2]
```diff
@@ -900,6 +900,34 @@ def mock_actions_with_response(*resp)
...
       end
     end
+
+    context "when using http protocol" do
+      context "when using ssl without cert verification" do
+        protocol = "http"
+        subject do
+          require "logstash/outputs/elasticsearch"
+          settings = {
+            "protocol" => protocol,
```
Comment (talevy) [inline spec/outputs/elasticsearch_spec.rb]: why not hardcode `http` here if no other is tested, and it is even mentioned in the higher level context

File: spec/outputs/elasticsearch_spec.rb [hunk 3]
```diff
@@ -900,6 +900,34 @@ def mock_actions_with_response(*resp)
         end
       end
     end
+
+    context "when using http protocol" do
+      context "when using ssl without cert verification" do
```
Comment (talevy) [inline spec/outputs/elasticsearch_spec.rb]: is having a separate context within a context necessary?

Comment (jsvd) [inline spec/outputs/elasticsearch_spec.rb]: not necessary but better categorizes the test. This is all in the hopes that more tests will come and the outer context will have other contexts/tests.

File: lib/logstash/outputs/elasticsearch.rb (hunk 1 above)
Comment (jsvd) [inline lib/logstash/outputs/elasticsearch.rb]: oops missed that, well spotted

**Comment by talevy** [comment]
LGTM

**Comment by talevy** [comment]
jenkins, please test this.

**Comment by talevy** [comment]
please test this, jenkins.

**Comment by talevy** [comment]
please jenkins, test this.

**Comment by talevy** [comment]
please jenkins test this

**Pull request closed without merging**